optaplanner-docs/src/main/asciidoc/QuickStart/VaccinationScheduler/VaccinationSchedulerQuickStart-chapter.adoc

```diff
@@ -0,0 +1,275 @@
+
+:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+//:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
```

Comment:
(soft) In the builds, we use the variable "productized". any chance we can name it like that?